xapi.spec.in

```diff
@@ -222,6 +222,8 @@ rm -rf $RPM_BUILD_ROOT
 /etc/xapi.d/plugins/install-supp-pack
 /etc/xapi.d/plugins/disk-space
 /etc/xapi.d/extensions
+/etc/xapi.d/xapi-logrotate.conf
+%attr(0755,root,root) /etc/xapi.d/xapi-logrotate.sh
```

Comment:
I think this should probably be %{_libexecdir}/%{name}/xapi-logrotate.sh, as described 
[in the packaging guidelines.](https://fedoraproject.org/wiki/Packaging:Guidelines#Libexecdir)
You may find that you don't need the %attr specification if you put it there, as long as it has the right attributes when installed by the build script.